inspire/ext/formatter_jinja_filters/record.py

```diff
@@ -19,6 +19,7 @@
 
 from inspire.ext.formatter_jinja_filters.general import apply_template_on_array
 from inspire.utils.bibtex import Bibtex
+from inspire.utils.latex import Latex
```

Comment:
hmm, what happened to this file? You import `Latex` but never use it. And you removed the Bibtex filter.

Even if for the moment you are only using the formats via AJAX, leave both as a jinja2 filters as well in case we need them.